release-notes/opensearch-observability.release-notes-1.3.0.0.md

```diff
@@ -0,0 +1,95 @@
+* fix interval selector issue, revert interval function changes ([#563](https://github.com/opensearch-project/observability/pull/563))
+* remove bold letter and extra pranthesis ([#559](https://github.com/opensearch-project/observability/pull/559))
+* Issue horizontal bar ([#556](https://github.com/opensearch-project/observability/pull/556))
+* final live tail fixes ([#558](https://github.com/opensearch-project/observability/pull/558))
+* fix page flicker for live tail ([#541](https://github.com/opensearch-project/observability/pull/541))
+* Fix multiple flyouts issue in explorer ([#538](https://github.com/opensearch-project/observability/pull/538))
+* Flyout bugs ([#540](https://github.com/opensearch-project/observability/pull/540))
+* detete request and response changes for event and panels ([#530](https://github.com/opensearch-project/observability/pull/530))
+* Issue/darkmode support viz config ([#521](https://github.com/opensearch-project/observability/pull/521))
+* Visualizations do not follow set timerange ([#516](https://github.com/opensearch-project/observability/pull/516))
+* Fix empty userConfigs stringify ([#513](https://github.com/opensearch-project/observability/pull/513))
+* Fix lower margin of autocomplete being cut off ([#512](https://github.com/opensearch-project/observability/pull/512))
+* fix issue of clicking query caused crash ([#515](https://github.com/opensearch-project/observability/pull/515))
+* Feature viz saving on missing fields ([#511](https://github.com/opensearch-project/observability/pull/511))
+* Fix events flyout bugs and Styling ([#510](https://github.com/opensearch-project/observability/pull/510))
+* Revert query pre-processing for parse command ([#497](https://github.com/opensearch-project/observability/pull/497))
+* Fix create/edit page bug ([#475](https://github.com/opensearch-project/observability/pull/475))
+* Fix queries being filtered out ([#472](https://github.com/opensearch-project/observability/pull/472))
+
+### Documentation
+* Sync PPL commands doc with main repo ([#549](https://github.com/opensearch-project/observability/pull/549))
+* Fixed documentation links ([#534](https://github.com/opensearch-project/observability/pull/534))
+* Add parse command docs ([#535](https://github.com/opensearch-project/observability/pull/535))
+* Update Release Notes 1.2.4 ([#370](https://github.com/opensearch-project/observability/pull/370))
+* updating readme and badges ([#352](https://github.com/opensearch-project/observability/pull/352))
+
+### Maintenance
+* Change Default CI java version to 11 ([#504](https://github.com/opensearch-project/observability/pull/504))
+* Update backport and add auto-delete workflows ([#496](https://github.com/opensearch-project/observability/pull/496))
+* Add auto backporting functionality ([#491](https://github.com/opensearch-project/observability/pull/491))
+* bumping version to 1.2.2 ([#346](https://github.com/opensearch-project/observability/pull/346))
```

Comment:
these are from previous releases, could you remove them for now? PR 346, 360, 370

i think they got included because they were backported to 1.2 branch, which creates new commits, instead of cutting 1.2 from main to include these commits, so workflow thinks that they were not included in previous release. Not sure how to fix it right now because of our current branching strategy which uses backports